### ISSUE TITLE
Dependabot opens pull requests without prefix

### ISSUE THREAD

**Issue by localheinz** [dependabot/dependabot-core#3245]
**Package manager/ecosystem**

composer

**Manifest contents prior to update**

not relevant

**Updated dependency**

not relevant

**What you expected to see, versus what you actually saw**

### Two days ago

Dependabot opened pull requests with configured prefix, and, in particular, with the platform prefix:

![CleanShot 2021-03-08 at 11 18 47](https://user-images.githubusercontent.com/605483/110308195-1f64ff00-8000-11eb-8d3f-914738b4d702.png)

### Today

Dependabot opened pull requests without the configured prefix, and, in particular, without the platform prefix:

![CleanShot 2021-03-08 at 11 16 04](https://user-images.githubusercontent.com/605483/110308057-edec3380-7fff-11eb-9739-98fe2a4c9926.png)

### Configuration

Both projects share the same configuration for Dependabot.

```yaml
# https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates

version: 2

updates:
  - allow:
      - dependency-type: "development"
    commit-message:
      include: "scope"
      prefix: "composer"
    directory: "/"
    labels:
      - "dependency"
    open-pull-requests-limit: 10
    package-ecosystem: "composer"
    schedule:
      interval: "daily"
    versioning-strategy: "increase"

  - commit-message:
      include: "scope"
      prefix: "github-actions"
    directory: "/"
    labels:
      - "dependency"
    open-pull-requests-limit: 10
    package-ecosystem: "github-actions"
    schedule:
      interval: "daily"
```

**Comment by NoNameProvided** [comment]
This happens for JS dependencies too.

**Comment by thepwagner** [comment]
👋 confirming this bug: a testing cohort of Dependabot users followed a new code path to create PullRequest/commit messages - and it affected these settings.

Sorry for the impact on your workflow(s), we've rolled back the new path for now.

**Comment by thepwagner** [comment]
The regression was addressed in ` 0.136.0-1d13f180215dbdd5052d1501732ca89c86e58bf0` (since `2021-03-08 17:06:51 +0000`).

For further context: it only affected repositories that provided partial commit options (e.g. `prefix` without `prefix-development`). Dependabot would incorrectly assume a `nil` value was the same as an empty string.
